app/views/contacts/success.html.erb

```diff
@@ -1,9 +1,11 @@
 <turbo-frame id="contact-form">
-  <h2>Help request successfully sent</h2>
-  <p class="mt-3">Your message has been sent to the SDR team. We will respond to you soon.</p>
+  <%= render Elements::AlertComponent.new(variant: :success, dismissible: !@welcome) do %>
+    <div class="h2">Help request successfully sent</div>
```

Comment:
Can you say more about changing from an `h2` to a `div`?

Reply:
This is an H2 display, not semantically a heading.